test/nodes/test_prompt_node.py

```diff
@@ -532,7 +532,7 @@ def test_complex_pipeline_with_shared_prompt_model_and_prompt_template_yaml(tmp_
               params:
                 model_name_or_path: google/flan-t5-small
                 model_kwargs:
-                  torch_dtype: torch.bfloat16
+                  torch_dtype: auto
```

Comment:
I agree these tests are heavy, but I believe we should duplicate this test to explicitly test for `auto` and explicit references separately.